src/createScaffoldMiddleware.ts

```diff
@@ -0,0 +1,30 @@
+import { JsonRpcMiddleware, JsonRpcSuccess } from './JsonRpcEngine';
+
+type Serializable =
```

Comment:
REQ: We should remove unknowns here, as they allow more than what we would like for this type.

```ts
type BasicSerializable =
  | boolean
  | number
  | string
  | null
  | undefined;

type Serializable =
  | BasicSerializable
  | Record<string, BasicSerializable>
  | BasicSerializable[];
```

As originally written `{ k: () => { console.log('hi, Erik!') }` would be allowed, as would `[Symbol(🔢 )]`

Reply:
Good point! I did one better and defined a recursive `Json` type: 00fae51